public/styles/components/form-fields/_image-select.scss

```diff
@@ -43,6 +43,15 @@
     }
   }
 
+  &__replace-button {
+    margin-right: 20px;
```

Comment:
that's one way to do it. But if there is no button on the right, you're left with a right margin that might break your layout. What you can do instead is have a left margin on any button that follows another button:
```
button + button {
  margin-left: 20px;
}
```